src/makeRobot.js

```diff
@@ -39,6 +39,72 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this
+        .version}, wheels: ${this.wheels}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    goForward(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+
+      this.coords.y += step;
+
+      return this;
```

Comment:
метод можнеа скоротити, зміни так кожний метод в об'єкті де потрібно
```suggestion
      if (step > 0) {
        this.coords.y += step;
      }

      return this;
```